### PR TITLE
Add test/extended/operators OWNERS file

### PR DESCRIPTION
**Pull request by kevinrizza**

### CODE REVIEW

File: test/extended/operators/OWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - ecordell
+approvers:
+  - smarterclayton
+  - bparees
+  - mfojtik
+  - knobunc
+  - ironcladlou
+  - adambkaplan
+  - kevinrizza
+  - ecordell
```
Comment (bparees) [inline test/extended/operators/OWNERS:20]: you should only list OLM people here.

it's inherited, so i'll still have permissions from the top level files.

**Comment by kevinrizza** [comment]
@bparees updated. please take another look

**Pull request merged**